ros_ign_bridge/src/factories.cpp

```diff
@@ -335,6 +346,30 @@ get_factory_impl(
       >
     >("sensor_msgs/BatteryState", ign_type_name);
   }
+  if (
+    (ros_type_name == "visualization_msgs/Marker" || ros_type_name == "") &&
+     ign_type_name == "ignition.msgs.Marker")
+  {
+    ROS_ERROR_STREAM("visualization_msgs/Marker");
+    return std::make_shared<
+      Factory<
+        visualization_msgs::Marker,
+        ignition::msgs::Marker
+      >
+    >("visualization_msgs/Marker", ign_type_name);
+  }
+  if (
+    (ros_type_name == "visualization_msgs/MarkerArray" || ros_type_name == "") &&
+     ign_type_name == "ignition.msgs.Marker_V")
+  {
+    ROS_ERROR_STREAM("visualization_msgs/MarkerArray");
```

Comment:
Should this error stream message be here?

Reply:
Nope, removed.